tests/Rules/Function/IncludeToRender/IncludeToRenderFunctionRule.fixed.twig

```diff
@@ -0,0 +1,16 @@
+{{ render('template.html', _context) }}
+{{ render('template.html', {'foo': 'bar'}|merge(_context)) }}
+{{ render('template.html', {'foo': 'bar'}|merge(_context), true) }}
+{{ render('template.html', {'foo': 'bar'}|merge(_context), true, true) }}
+{{ render('template.html', with_context = true) }}
```

Comment:
If I understand the render method, there is no `with_context` param.

Reply:
Yes fixed it and added with_context = false.

Reply:
The changes are still in discussion but wanted to show example how fixer could make a upgrade possible.